sensor_msgs/Image src_image
---
bool success
sensor_msgs/Image dst_image
